src/control/server/storage/scm/mocks.go

```diff
@@ -45,6 +45,36 @@ type (
 	}
 )
 
+// MockDiscovery returns a mock SCM module of type exported from ipmctl.
+func MockDiscovery() ipmctl.DeviceDiscovery {
```

Comment:
We really don't want any ipmctl stuff to be exported/exposed beyond this package's boundaries. Keeping this in a _test.go means that it can only be used by tests in this package.

Reply:
ok will move it back